src/nemos/glm.py

```diff
@@ -698,6 +760,21 @@ def simulate(
         ValueError
             - If the instance has not been previously fitted.
 
+        Examples
+        --------
+        >>> # example input
+        >>> import numpy as np
+        >>> X, y = np.random.normal(size=(10, 2)), np.random.uniform(size=10)
+        >>> Xnew = np.random.normal(size=(20, ) + X.shape[1:])
+
+        >>> # define and fit model
+        >>> import nemos as nmo
+        >>> model = nmo.glm.GLM()
+        >>> model = model.fit(X, y)
+
+        >>> # generate spikes and rates
+        >>> random_key = jax.random.key(123)
+        >>> spikes, rates = model.simulate(random_key, Xnew)
```

Comment:
```suggestion

        >>> # define and fit model
        >>> import nemos as nmo
        >>> model = nmo.glm.GLM()
        >>> model = model.fit(X, y)

        >>> # generate spikes and rates
        >>> Xnew = np.random.normal(size=(20, X.shape[1]))
        >>> random_key = jax.random.key(123)
        >>> spikes, rates = model.simulate(random_key, Xnew)
```
same point about Xnew